siliconcompiler/report/summary_table.py

```diff
@@ -69,7 +69,7 @@ def _show_summary_table(chip, flow, flowgraph_nodes, show_all_indices):
 
         libraries = set()
         for val, step, _ in chip.schema._getvals('asic', 'logiclib'):
-            if not step or step in flowgraph_nodes:
+            if not step or (step, index) in flowgraph_nodes:
```

Comment:
Watch out, also need to add in `index` to the for loop (otherwise this is using a stale value of index I believe)

Reply:
Thanks for catching that one.